manyfews/calculations/tasks.py

```diff
@@ -15,3 +20,19 @@ def hello_celery():
     This is an example of a task that can be scheduled via celery.
     """
     logger.info("Hello logging from celery!")
+
+
+# prepare GEFS data
+dt = float(settings.GEFS_TIME_STEP)
+forecastDays = int(settings.GEFS_FORECAST_DAYS)
+dataBaseWriter(dt=dt, forecastDays=forecastDays)
+
```

Comment:
This should be a function, like `hello_celery` (which you could probably get rid of), with a `shared_task` annotation.